io.github.Loganavter.Improve-ImgSLI.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.Loganavter.Improve-ImgSLI
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.8'
+command: Improve-ImgSLI
+tags: ["stable"]
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --share=ipc
+  - --device=dri
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/man
+  - /share/pkgconfig
+  - '*.la'
+  - '*.a'
+
+modules:
+  - python3-modules.json
+  - name: Improve-ImgSLI
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin
+      - install -d ${FLATPAK_DEST}/lib/Improve-ImgSLI
+      - install -d ${FLATPAK_DEST}/share/applications
+      - install -d ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps
+      - install -d ${FLATPAK_DEST}/share/licenses/Improve-ImgSLI
+      - install -d ${FLATPAK_DEST}/share/metainfo
+      - install -d ${FLATPAK_DEST}/share/fonts/truetype/
+      - rm -f ${FLATPAK_DEST}/share/metainfo/com.riverbankcomputing.PyQt.BaseApp.metainfo.xml
+      - |
+        cat > ${FLATPAK_DEST}/bin/Improve-ImgSLI << EOF
+        #!/bin/sh
+        # Путь /app/ соответствует ${FLATPAK_DEST} во время выполнения
+        exec python3 /app/lib/Improve-ImgSLI/image_comparison_app.py "\$@"
+        EOF
+      - chmod +x ${FLATPAK_DEST}/bin/Improve-ImgSLI
+      - install -Dm644 clickable_label.py      -t ${FLATPAK_DEST}/lib/Improve-ImgSLI/
+      - install -Dm644 image_comparison_app.py -t ${FLATPAK_DEST}/lib/Improve-ImgSLI/
+      - install -Dm644 image_processing.py     -t ${FLATPAK_DEST}/lib/Improve-ImgSLI/
+      - install -Dm644 flag_icons.py           -t ${FLATPAK_DEST}/lib/Improve-ImgSLI/
+      - install -Dm644 translations.py         -t ${FLATPAK_DEST}/lib/Improve-ImgSLI/
+      - install -Dm644 SourceSans3-Regular.ttf -t ${FLATPAK_DEST}/share/fonts/truetype/
+      - sed -i 's|^Categories=.*|Categories=Graphics;Utility;Viewer;|' improve-imgsli.desktop
+      - sed -i "s|^Icon=.*|Icon=${FLATPAK_ID}|" improve-imgsli.desktop
```

Comment:
don't use sed to change the desktop file. There is a `desktop-file-edit` command for that.